### ISSUE TITLE
setDefaultMarkerWidth is missed in fixing-nft branch

### ISSUE THREAD

**Issue by kalwalt** [kalwalt/jsartoolkit5#19]
i saw that `setDefaultMarkerWidth` is missed in **fixing-nft** branch compared to the actual master **artoolkitx/jsartoolkit5** branch need to think if will add it again or not.

**Comment by kalwalt** [comment]
This function was added i think for the Type definitions see https://github.com/artoolkitx/jsartoolkit5/blob/b2a6c7529777999030773e02266799390482ba2a/js/artoolkit.api.d.ts#L69

**Comment by nicolocarpignoli** [comment]
yes, I think we can avoid this question for now, it does not seems crucial. :)

**Comment by kalwalt** [comment]
We can verify the typescript version in another moment. ALso i have no experience in this sense...

**Comment by kalwalt** [comment]
@ThorstenBux Sorry to re-open, but it seems that this is required by arStudio see this commit https://github.com/augmentmy-world/arStudio/pull/2/commits/3ffb20305ffbc91b0a7ecc61a9a9e005472ca459
